doc/classes/@GlobalScope.xml

```diff
@@ -2530,61 +2530,70 @@
 			The maximum number of game controller axes: OpenVR supports up to 5 Joysticks making a total of 10 axes.
 		</constant>
 		<constant name="MIDI_MESSAGE_NONE" value="0" enum="MIDIMessage">
-			Enum value which doesn't correspond to any MIDI message. This is used to initialize [enum MIDIMessage] properties with a generic state.
+			Does not correspond to any MIDI message. This is the default value of [member InputEventMIDI.message].
 		</constant>
 		<constant name="MIDI_MESSAGE_NOTE_OFF" value="8" enum="MIDIMessage">
-			MIDI note OFF message. Not all MIDI devices send this event; some send [constant MIDI_MESSAGE_NOTE_ON] with zero velocity instead. See the documentation of [InputEventMIDI] for information of how to use MIDI inputs.
+			MIDI message sent when a note is released.
+			[b]Note:[/b] Not all MIDI devices send this message; some may send [constant MIDI_MESSAGE_NOTE_ON] with [member InputEventMIDI.velocity] set to [code]0[/code].
 		</constant>
 		<constant name="MIDI_MESSAGE_NOTE_ON" value="9" enum="MIDIMessage">
-			MIDI note ON message. Some MIDI devices send this event with velocity zero instead of [constant MIDI_MESSAGE_NOTE_OFF], but implementations vary. See the documentation of [InputEventMIDI] for information of how to use MIDI inputs.
+			MIDI message sent when a note is pressed.
 		</constant>
 		<constant name="MIDI_MESSAGE_AFTERTOUCH" value="10" enum="MIDIMessage">
-			MIDI aftertouch message. This message is most often sent by pressing down on the key after it "bottoms out".
+			?? MIDI aftertouch message. Sent to indicate a change in pressure while a note is being pressed down.
+			?? MIDI message sent to indicate a change in pressure while a note is being pressed down, also called aftertouch.
```

Comment:
Applies for these two, and more.
I am not sure whether to keep the introductory statement or not.
That is, "MIDI aftertouch message. Sent to [...]" or simplying "MIDI message sent to [...]"